front/src/stories/RollingStockSelector.stories.tsx

```diff
@@ -16,35 +14,26 @@ const Template: ComponentStory<typeof RollingStockSelector> = (args) => (
     <RollingStockSelector {...args} />
   </div>
 );
-
-const image = (
-  <>
-    <img src={exampleRollingStockImage1} alt="rollingStockHead" />
-    <img src={exampleRollingStockImage2} alt="rollingStockTail" />
-  </>
-);
-
-export const EmptyRollingStockSelector = Template.bind({});
-EmptyRollingStockSelector.args = {
-  choice: 'Choose a rollingstock',
+const defaultArgs = {
+  rollingStockSelected: ROLLING_STOCK_SAMPLE_DATA,
+  rollingStockComfort: 'STANDARD' as RollingStockComfortType,
 };
 
 export const NotCondensedRollingStockSelector = Template.bind({});
 NotCondensedRollingStockSelector.args = {
+  ...defaultArgs,
   condensed: false,
-  rollingStockComfort: 'HEATING',
-  rollingStockSelected: OSRD_ROLLINGSTOCKSELECTED_SAMPLE_DATA as RollingStock,
-  image,
-  comfort: 'Comfort',
-  comfortType: 'Heating',
 };
 
 export const CondensedRollingStockSelector = Template.bind({});
 CondensedRollingStockSelector.args = {
+  ...defaultArgs,
+  condensed: true,
+};
```

Comment:
In storybook, the selectedRollingStock doesn't seem to be taken into account. The selector is empty.